UnitystationLauncher/ContentScanning/MType.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace UnitystationLauncher.ContentScanning;
+
+
+public record MType
+{
+    public virtual bool WhitelistEquals(MType other)
+    {
+        return false;
```

Comment:
Is this supposed to check something here?

Reply:
Looks like it gets overridden in other Classes is ss14 code